PROJECT

```diff
@@ -3,4 +3,14 @@ layout:
 - go.kubebuilder.io/v3
 projectName: multi-cluster-traffic-controller
 repo: github.com/Kuadrant/multi-cluster-traffic-controller
+resources:
+- api:
+    crdVersion: v1
+    namespaced: true
+  controller: true
+  domain: kuadrant.io
+  group: networking.k8s.io
+  kind: Ingress
+  path: github.com/Kuadrant/multi-cluster-traffic-controller/api/v1
+  version: v1
```

Comment:
I'm not sure any of this should be added since we don't provide that api?

Reply:
nice spot. Removed.